vars/other.groovy

```diff
@@ -29,3 +29,32 @@ def get_username() {
   else
     return cause.userId;
 }
+
+def parse_url(failed_job, is_stage=false) {
+  def tokens = failed_job.split('/')
+  if (is_stage) {
+    return tokens[-2]
+  }
+  return tokens[-2] + "/" + tokens[-1]
+}
+
+def get_team_email(url) {
+  if(url.contains('cfme')) {
+    return "rhevm-qe-cfme@redhat.com"
+  }
+  if(url.contains('infra')){
+    return "rhv-qe-devops@redhat.com"
```

Comment:
You can remove infra and return it at the end in else block or just at the end as default return